python/test/gluon/test_lowerings.py

```diff
@@ -255,6 +254,18 @@ def kernel(x_ptr, z_ptr, M: ttgl.constexpr, B: ttgl.constexpr, src_layout: ttgl.
         z_offs = ttgl.arange(0, B, layout=dst_layout)
         ttgl.store(z_ptr + z_offs, h)
 
+    if src_layout == "linear_layout":
+        if M == 32:
+            src_layout = ttgl.DistributedLinearLayout(
+                reg_bases=[],
+                lane_bases=[[0], [16], [4], [2], [1]] + [[0]] * (THREADS_PER_WARP >> 6),
```

Comment:
This should be `log2(THREADS_PER_WARP / 32)`. With this, you can probably put it directly in the `parametrize` list.

Reply:
#8016